### PR TITLE
Feature/headplot

### PR DESCRIPTION
**Pull request by Ho-ChunHuang-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by Ho-ChunHuang-NOAA** [comment]
this branch
(1) Add headline plot for ozmax8 and pmave; out in headline.$VDATE.
(2) fixed the missing obar in the ozone fbar_obar plot
(3) Add options of PMAVE threshold value in settings.py that allows CSI=">35" now appears in the performance diagram.
In fact, I added options of '>35, >40, >45' for future consideration.

headline plots (1) :;  can be seen at https://www.emc.ncep.noaa.gov/mmb/hchuang/ftp/ or /lfs/h2/emc/ptmp/ho-chun.huang/evs/v1.0/plots/aqm/headline.20230724
(2) and (3) can be seen at https://www.emc.ncep.noaa.gov/users/verification/air_quality/aqm/evs/

**Comment by PerryShafran-NOAA** [comment]
Hi, @Ho-ChunHuang-NOAA .  I will be starting this PR.  Just wanted to confirm, only plots needs to be tested for this PR, not prep or stats.  

Thanks!

Perry

**Comment by PerryShafran-NOAA** [comment]
Hi, @Ho-ChunHuang-NOAA ,

I have completed the plots job.  You can find the .o file here:/ lfs/h2/emc/vpppg/noscrub/perry.shafran/pr176test/EVS/dev/drivers/scripts/aqm/plots/jevs_aqm_grid2obs_plots_00.o73178066

and the plot files are here: /lfs/h2/emc/ptmp/perry.shafran/evs/v1.0/plots/aqm

Please validate that the job and all plots are what you expect to see.

Thanks!

Perry

**Comment by Ho-ChunHuang-NOAA** [comment]
Hi, Perry:

The figures look exactly as I expected for both regular (atmos.20230724)
and headline plot (headline.20230724).

Ho-Chun

--

Ho-Chun Huang, Ph.D.

Physical Scientist III, Contractor with Lynker in Support of

NOAA/NWS/NCEP/EMC, U.S. Department of Commerce

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958

**Comment by PerryShafran-NOAA** [comment]
Excellent!  Let me do a code check.

Perry

File: dev/drivers/scripts/aqm/plots/test_jevs_aqm_plots.sh
Comment (PerryShafran-NOAA) [inline dev/drivers/scripts/aqm/plots/test_jevs_aqm_plots.sh:1]: Hi, Ho-Chun,

We need to have the official file called "jevs_aqm_grid2obs_plots.sh".  Can you make sure that all the changes are in the correct file and the test script removed?

Thanks!

Perry

File: dev/drivers/scripts/aqm/plots/test_jevs_aqm_plots.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+export NET=evs
+export STEP=plots
+export COMPONENT=aqm
+export RUN=atmos
+export VERIF_CASE=grid2obs
+export MODELNAME=aqm
+export modsys=aqm
+export mod_ver=${aqm}
+
+export MET_bin_exec=bin
```
Comment (PerryShafran-NOAA) [inline dev/drivers/scripts/aqm/plots/test_jevs_aqm_plots.sh]: This line is not needed as it is taken care of in machine.conf.

Perry

**Comment by PerryShafran-NOAA** [comment]
Also, please be sure to merge in the current develop branch.

Perry

**Comment by Ho-ChunHuang-NOAA** [comment]
Hi, Perry:

I removed the testing file and  synced with the latest develop branch.

Ho-Chun

--

Ho-Chun Huang, Ph.D.

Physical Scientist III, Contractor with Lynker in Support of

NOAA/NWS/NCEP/EMC, U.S. Department of Commerce

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958

**Comment by PerryShafran-NOAA** [comment]
Great! Thanks!

Perry

**Comment by PerryShafran-NOAA** [comment]
Looks great, I will merge now.

Perry

**Review by PerryShafran-NOAA — APPROVED** [review]

**Pull request merged**